packages/module/src/Severity/Severity.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useMemo } from 'react';
+import clsx from 'clsx';
+import {
+  SeverityCriticalIcon,
+  SeverityImportantIcon,
+  SeverityMinorIcon,
+  SeverityModerateIcon,
+  SeverityNoneIcon,
+  SeverityUndefinedIcon
+} from '@patternfly/react-icons';
+import { createUseStyles } from 'react-jss';
+
+const severityCritical = {
+  '& svg': {
+    '& path': { fill: 'var(--pf-t--global--color--status--danger--default)' }
+  }
+};
+
+const severityImportant = {
+  '& svg': {
+    '& path': { fill: 'var(--pf-t--global--icon--color--severity--important--default)' }
+  }
+};
+
+const severityModerate = {
+  '& svg': {
+    '& path': { fill: 'var(-pf-t--global--icon--color--severity--moderate--default)' }
+  }
+};
+
+const severityMinor = {
+  '& svg': {
+    '& path': { fill: 'var(--pf-t--global--icon--color--severity--minor--default)' }
+  }
+};
+
+const severityNone = {
+  '& svg': {
+    '& path': { fill: 'var(--pf-t--global--icon--color--severity--none--default)' }
+  }
+};
+
+const severityUndefined = {
+  '& svg': {
+    '& path': { stroke: 'var(--pf-t--global--icon--color--severity--undefined--default)' }
+  }
+};
+
+const useStyles = createUseStyles({
+  severity: {
+    display: 'inline-block',
+    'line-height': 0,
+    '& svg': {
+      position: 'relative',
+      top: 'var(--pf-t--global--spacer--sm)',
+      height: 'calc(var(--pf-t--global--spacer--md) * 1.75)',
+    },
+  },
+  severityLabel: {
+    'margin-left': 'var(--pf-t--global--spacer--xs)'
+  },
```

Comment:
These styles are interesting! There may be a reason for them to be the way they are, but I am curious about the use of `top` and the `height`, particularly the height's `calc()`. My guess is this might have been needed for the old battery design and might not be needed still?

Also we have a ["text-to-element" semantic spacer](https://github.com/patternfly/patternfly/blob/b93029abee9117da890c05f219412e3c423d57b4/src/patternfly/base/tokens/tokens-default.scss#L497C3-L497C56) we use with icons + text, but it expects there to be no space (from whitespace/newlines) in the layout between the icon and text, which is ideally used in a flex/grid layout since those will remove the space between items in the layout. I'm wondering if using a flex layout for the parent of the icon + text pair might be a better way of aligning/styling the icon instead of the `line-height`, `top`, and `height` currently used.

Reply:
Thanks @mcoker.  Yes these were left over from the battery work, they need to be cleaned up.  Thanks for the review I'll go through and make your recommended updates then ping you once they are updated.